### PR TITLE
[CRT-3670] Unity Plugin Compatibility

### PR DESCRIPTION
**Pull request by Lunar2kPS**
## What Has Changed
- Updated the codebase to work with our LKG Unity Plugin on Unity 2021.3 (C# 9),
    - Removed raw string literals prefixed with `$$"""` (replaced them with interpolated verbatim strings prefixed with `$@"`)
    - Removed underscores from namespace names (ToolkitAPI, ToolkitCLI, etc.)
- Renamed Display to TKDisplay
- Renamed DisplayInfo to TKDisplayInfo
- Added C#-XML /// docs comments to various classes, methods, and fields

## Notes
- This PR is related to [UnityPlugin PR #187](https://github.com/Looking-Glass/UnityPlugin/pull/187)
    - For UnityPlugin workflow, note that we now have LKG Toolkit as a Git submodule in the UnityPlugin (OUTSIDE of the Unity project), and we have a Bash script under `/scripts/update-lkg-toolkit.sh` that has a fairly thorough comment documenting the script and its usage/limitations so far, at the top of the file!

### CODE REVIEW

File: Toolkit-API/Bridge/JsonHelpers.cs
Comment (NullandKale) [inline Toolkit-API/Bridge/JsonHelpers.cs:1]: I would move this to a Utils.cs class or if its only used in a single file I would just move this function to that file.

**Review by NullandKale — COMMENTED** [review]
Looks good to me!

File: Toolkit-API/Bridge/JsonHelpers.cs
Comment (Lunar2kPS) [reply Toolkit-API/Bridge/JsonHelpers.cs:1]: It was used by 2 classes I think it was Calibration.cs and DefaultQuilt.cs,
So would you like maybe `Toolkit-API/Utils.cs`?
I'd probably make it (personally) `Toolkit-API/Utility/JsonUtility.cs`, but Unity has a `JsonUtility`, so I named it JsonHelpers instead haha.
But completely up to you, I don't mind anything with regards to this

**Comment by Lunar2kPS** [comment]
Note that I didn't add any async support for now, and it's not a blocker for us currently (unless it just so happens to be involved with our 2sec+ performance issues), but it is happily welcomed! ;) (and not saying it should be for this PR necessarily)

**Comment by NullandKale** [comment]
Merged!

**Pull request merged**